specs/GetFavicon.md

```diff
@@ -0,0 +1,149 @@
+# Background
+A Favicon is an asset which is a part of a webpage, and typically displayed on each tab. Developers would
+like to have an API which allows them to retrieve the Favicon of a webpage, if it has been set, as well as get an update whenever the favicon has changed.
+
+# Description
+We propose a new Webview2 event which would allow developers access the current Favicon of a page, 
+as well as be notified when the favicon changes. This means when a page first loads, it would fire
+the Favicon change event as the icon has inialized to null. DOM or Javascript may change the Favicon,
```

Comment:
```suggestion
the FaviconChanged event since before parsing the HTML document there is no known favicon. DOM or JavaScript may change the Favicon,
```